install.sh

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env sh
+set -eu
+(set -o pipefail 2>/dev/null) && set -o pipefail
+
+help() {
+  cat <<'EOF'
+Install spaces binaries (spaced and space-cli) from GitHub releases.
+
+USAGE:
+    install.sh [options]
+
+FLAGS:
+    -h, --help      Display this message
+    -f, --force     Force overwriting existing binaries
+    -v, --verbose   Show detailed output
+
+OPTIONS:
+    --tag VERSION   Specific version to install (e.g., 0.0.6a), defaults to latest release
+    --to LOCATION   Where to install the binaries (default: auto-detected based on OS)
+                   For macOS: ~/.local/bin if exists, else ~/bin
+                   For Linux: ~/.local/bin
+
+NOTES:
+    The script will automatically choose an appropriate user-writable location.
+    The chosen directory will be added to your PATH if needed.
+EOF
+}
+
+say() {
+  echo "install: $*" >&2
+}
+
+err() {
+  if [ -n "${td-}" ]; then
+    rm -rf "$td"
+  fi
+  say "error: $*"
+  say "run './install.sh --help' to see available options"
+  exit 1
+}
+
+need() {
+  if ! command -v "$1" > /dev/null 2>&1; then
+    err "need $1 (command not found)"
+  fi
+}
+
+add_to_path() {
+  local install_dir="$1"
+  local shell_rc
+  local reload_needed=false
+
+  # Detect shell configuration file
+  if [ -n "${ZSH_VERSION-}" ]; then
```

Comment:
$ZSH_VERSION is only set when you're actually running zsh as your interactive shell but the script executes in sh/bash env. Maybe use `$SHELL` (returns `/bin/zsh` for me) or just check if its macos.